pages/index.vue

```diff
@@ -1609,46 +1487,19 @@ export default {
     },
     setRouteQueryState() {
       const flat = (key) => (this[key] !== "" ? `${key}=${this[key]}&` : "")
-      const deep = (key) => {
-        const haveItems = [...this[key]].length
-        if (haveItems && this[key]["value"] !== "") {
-          // Exclude files fro  query params
-          const filesRemoved = this[key].filter((item) => !(item?.value?.[0] instanceof File))
-          return `${key}=${JSON.stringify(filesRemoved)}&`
-        }
-        return ""
-      }
-      let flats = [
-        "method",
-        "url",
-        "path",
-        !this.URL_EXCLUDES.auth ? "auth" : null,
-        !this.URL_EXCLUDES.httpUser ? "httpUser" : null,
-        !this.URL_EXCLUDES.httpPassword ? "httpPassword" : null,
-        !this.URL_EXCLUDES.bearerToken ? "bearerToken" : null,
-        "contentType",
-      ]
-        .filter((item) => item !== null)
-        .map((item) => flat(item))
-      const deeps = ["headers", "params"].map((item) => deep(item))
-      const bodyParams = this.rawInput ? [flat("rawParams")] : [deep("bodyParams")]
+      let flats = ["url", "path"].filter((item) => item !== null).map((item) => flat(item))
+      const request = "request=" + JSON.stringify(this.request)
```

Comment:
I'm not a fan of this implementation - stringified JSON object will have multiple closing tags in it. Which produce these kinda syntax errors breaking populating parameter data to state / ui.

**error:**

<img width="412" alt="Screenshot 2021-03-28 at 11 53 04 PM" src="https://user-images.githubusercontent.com/10395817/112763127-ffe64400-8fb7-11eb-92c9-dab61ba5ef3b.png">

**to reproduce:**
1. visit [deploy preview](https://deploy-preview-1567--hoppscotch.netlify.app)
2. make a request setup other than the default one
3. copy url
4. reset state or open new incognito window
5. goto copied url